ALTA-Gurumu/QE-Web
# Automation Test Beranda Page#14

- Add landing page object
- Add Beranda Page Object
- Test Functionality Cari Guru
- Verify Beranda Page
- Field Validation Beranda Page
- Delete /target
